docs/project/2024/carina.grec/index.md

```diff
@@ -0,0 +1,115 @@
+# House Thermometer
+
+The House Thermometer project uses a Raspberry Pi Pico W to measure indoor temperatures, visually display data and enable remote monitoring and data analysis.
+
+:::info 
+
+**Author**: Grec Carina-Gabriela \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-carinagrec
+
+:::
+
+## Description
+
+House thermometer capable of displaying the current room temperature on an LCD display for immediate local access, as well as on a web application for remote monitoring. The system uses a digital temperature sensor for accurate measurements and features an RGB LED that changes color based on temperature ranges, enhancing visual feedback. Also, it has an external memory to keep track of the temperatures from the last hours and display the average value. 
+
+## Motivation
+
+I chose this project because it seems like a pretty practical idea in the day-to-day life that can be implemented. It's a mix from multiple topics that we studied and i can also implement extra knowledge such as the EEPROM
+
+## Architecture 
+
+# Connectivity and Interactions
+
+1. **Sensor Interface → Data Processing and Logic Unit:**
+   - The sensor interface reads temperature data.
+   - This data is then sent to the data processing unit, where it is logged and processed for further actions.
+
+2. **Data Processing and Logic Unit → Display Controller:**
+   - Processed data, including current temperatures and system statuses, is forwarded from the data processing unit to the display controller.
+   - The display controller provides real-time visualization of the data.
+
+3. **Data Processing and Logic Unit → LED Controller:**
+   - Based on the temperature data, the data processing unit issues commands to the LED controller.
+   - The LED controller adjusts the LED's color and brightness according to the temperature conditions.
+
+4. **Data Processing and Logic Unit ↔ Network Interface:**
+   - The network interface functions as a gateway, facilitating communication between the web server and the data processing unit.
+   - It allows the data processing unit to send temperature data to the web server and receive commands from the web application.
+
+5. **Network Interface ↔ Web Application:**
+   - The web application interacts with the network interface to transmit user commands and receive data for display.
+   - This includes adjusting settings, monitoring real-time temperature updates, and accessing historical data logs.
+
+
+![Architecture](architecture.png)
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Raspberry Pi Pico W - a compact microcontroller board featuring the RP2040 chip and built-in Wi-Fi connectivity. It serves as the brain of the project, managing data collection from the temperature sensor, driving the display, controlling the RGB LED, and handling network communications.
```

Comment:
Are you sure that this is displayed correctly? Did you use `npm start` to look at it?